Assets/Branch/Demo/Scripts/BranchDemo.cs

```diff
@@ -0,0 +1,246 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+using System.Collections.Generic;
+using System;
+
+public class BranchDemo : MonoBehaviour {
+	
+	public InputField inputShortLink;
+	public Text lblCreditsValue;
+	public Text lblInstallCountValue;
+	public Text lblBuyCountValue;
+	public Text lblIdentifyUser;
+	public GameObject mainPanel;
+	public GameObject creditsHistoryPanel;
+	public GameObject referralCodePanel;
+
+	void Start()
+	{
+		//init debug mode
+		Branch.setDebug();
+
+		//init Branch
+		Branch.initSession( (parameters, error) => {
+			if (error != null)
+			{
+				Debug.LogError("Branch Error: " + error);
+			}
+			else
+			{
+				Debug.Log("Branch initialization completed:");
+				foreach (var key in parameters.Keys)
```

Comment:
I'm not entirely sure what you mean. A real user would likely be looking for specific keys like this:
```
var pictureId;
if (parameters.ContainsKey('pictureId')) {
    pictureId = parameters['pictureId'];
}
```
They likely wouldn't iterate over the whole dictionary to print them all out ;) I meant that, for the sake of your example, just printing the serialized dictionary would probably be cleaner.